src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx.PlatformNotSupported.cs

```diff
@@ -453,6 +453,28 @@ public new abstract class X64 : Sse42.X64
         /// </summary>
         public static Vector128<double> ExtractVector128(Vector256<double> value, byte index) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// 32-bit:
+        /// __m128i _mm256_extractf128_si256 (__m256i a, const int imm8)
+        ///   VEXTRACTF128 xmm/m128, ymm, imm8
+        ///
+        /// 64-bit:
+        /// __m128i _mm256_extractf128_si256 (__m256i a, const int imm8)
+        ///   VEXTRACTF128 xmm/m128, ymm, imm8
```

Comment:
The format we use for Arm is:
```
__m128i _mm256_extractf128_si256 (__m256i a, const int imm8)
  x86: VEXTRACTF128 xmm/m128, ymm, imm8
  x64: VEXTRACTF128 xmm/m128, ymm, imm8
```

In this case its identical for both so we can just use the normal form instead:
```
__m128i _mm256_extractf128_si256 (__m256i a, const int imm8)
  VEXTRACTF128 xmm/m128, ymm, imm8
```